native/src/routes/disclaimer/Disclaimer.js

```diff
@@ -25,7 +25,7 @@ class Disclaimer extends React.Component<PropsType> {
                  navigation={navigation}
                  files={{}}
                  language={language}
-                 cityCode={city}
+                 resourceCacheUrl='http://localhost:8080'
```

Comment:
Why did you choose to hardcode it here? Shouldn't you rather pass nothing? Just seems weird.